docs/managing-documents/reading.md

```diff
@@ -114,6 +114,12 @@ print(docs)
 Please see [the application developer's guide](https://docs.marklogic.com/guide/rest-dev/documents#id_80116)
 for more information on reading documents.
 
+## Obtaining the raw HTTP response
+
+The `client.documents.read` method also accepts a `return_response` parameter. When that
```

Comment:
One small tweak, and I don't think I've been 100% consistent about this in the Python docs - methods/functions accept "arguments", not "parameters" (whereas the REST API accepts "parameters").